app/views/welcome/index.html.erb

```diff
@@ -5,6 +5,8 @@
   style="
     background-image: url('https://images.unsplash.com/photo-1491554150239-a9062e24de5c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1yZWxhdGVkfDEyfHx8ZW58MHx8fHw%3D&w=1000&q=80');
     height: 100vh;
+    background-size: cover;
```

Comment:
Is that is what you were talking about for getting the image to render correctly?  Good to know!